### DISCUSSION TITLE
vite ssr mismatch

### DISCUSSION THREAD

**Discussion by mythshuazi** [element-plus/element-plus#8135] [Q&A]
使用vite2 构建的 ssr 在使用某些组件：el-date-picker、el-time-picker、el-select 等组件遇到 [vue warn]Hydration children mismatch、Hydration node mismatch 等报错，是否跟组件内部使用的 Teleport 有关？该怎么解决呢
![img](https://user-images.githubusercontent.com/7909545/172113590-29cfc9f6-d63f-44a4-83cf-c0fa23d7a379.png)

**Comment by gitsheny** [comment]
请问是按需导入还是全局导入？
